Docs: spell checker update. (#352)
ckeditor/ckeditor4-docs: docs/features/spellcheck/README.md

    This is a commercial solution provided by our partner, [WebSpellChecker](https://webspellchecker.com/). You can report any issues in its [GitHub repository](https://github.com/WebSpellChecker/wproofreader).
</info-box>

> [WProofreader](https://webspellchecker.com/wsc-proofreader) is an innovative proofreading tool that combines the functionality of "spell check as you type" and "spell check in a dialog" in a modern UI. Spelling, punctuation and grammar suggestions are available on hover with no clicking needed.

Reviewer: ```suggestion
[WProofreader](https://webspellchecker.com/wsc-proofreader) is an innovative proofreading tool that combines the functionality of "spell check as you type" and "spell check in a dialog" in a modern UI. Spelling, punctuation and grammar suggestions are available on hover with no clicking needed or as a convenient dialog, both with additional in-place replacement suggestions.
```

{@img assets/img/wproofreader_01.png 730 Spelling and grammar checking with WProofreader in CKEditor 4 WYSIWYG editor}

> The proofreader badge in the bottom-right corner shows you the total number of mistakes detected. Hover an underlined word to display the proofreader suggestions for any of the spelling and grammar mistakes found. This hover card allows the user to employ the feature on the go.

Reviewer: ```suggestion
The proofreader badge in the bottom-right corner shows you the total number of mistakes detected. Hover an underlined word to display the proofreader suggestions for any of the spelling and grammar mistakes found. This hovercard allows the user to employ the feature on the go.
```

## Supported Languages

### Language support

> The most popular languages used with WProofreader include: American English, British English, Canadian English, Canadian French, French, German, Italian, Greek, Spanish, Finnish, Danish, Dutch, Portuguese, Swedish, Ukrainian, Norwegian Bokmål, Brazilian Portuguese. There are, however, over 160 languages altogether, available for download from the WebSpellChecker site. Grammar checking is available for over 20 languages.

Reviewer: ```suggestion
The most popular languages offered by WebSpellChecker solutions include: American English, British English, Canadian English, Canadian French, French, German, Italian, Greek, Spanish, Finnish, Danish, Dutch, Portuguese, Swedish, Ukrainian, Norwegian Bokmål, Brazilian Portuguese. There are, however, over 160 languages altogether, available for download from the WebSpellChecker site. Grammar checking is available for over 20 languages.
```

Here you can check the [full list of available languages](https://webspellchecker.com/additional-dictionaries/).

### Custom dictionaries

Apart from the language dictionaries, WebSpellChecker offers two specialized dictionaries: a **medical** and a **legal** one, available for an additional fee.

There are also two types of custom dictionaries available.

One is the **user dictionary** that can be expanded and will grow during the regular use of the feature simply by adding new words to the dictionary. This is a perfect solution for editors and writers working on specific content that may contain slang or professional jargon.

The other is the so-called **company-wide dictionary**. These premade dictionaries can be uploaded by the system administrators or CKEditor 5 integrators and can be made available company-wide, accessible for all users. This way all benefits of a user-generated dictionary can be shared among the team, making the proofreading process more structured and controlled.

### Multi-language Support

> For content in multiple languages, it is also possible to use WProofreader. The spell check feature can automatically detect the text language properly. It is enough to choose “Auto Detect” in the Language dropdown in the WProofreader settings. Suggestions for spelling and grammar (where available) will be displayed properly for each text fragment.

Reviewer: As a matter of fact, this is supported in both WProofreader and SCAYT.

Author: Let's remove the first sentence, then.

> For content in multiple languages, it is also possible to use WProofreader. The spell check feature can automatically detect the text language properly. It is enough to choose “Auto Detect” in the Language dropdown in the WProofreader settings. Suggestions for spelling and grammar (where available) will be displayed properly for each text fragment.

Author: ```suggestion
If yoy have content in multiple languages, the spell check feature can automatically detect the text language properly. It is enough to choose “Auto Detect” in the Language dropdown in the WProofreader settings. Suggestions for spelling and grammar (where available) will be displayed properly for each text fragment.
```
